suite-native/thp/src/components/ThpPairingInfoHelpButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+
+import { BottomSheet, Box, Button, IconButton, Text, VStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+
+export const ThpPairingInfoHelpButton = () => {
+    const [isBottomSheetVisible, setIsBottomSheetVisible] = useState(false);
+
+    const toggleBottomSheet = () => setIsBottomSheetVisible(!isBottomSheetVisible);
+
+    return (
+        <Box>
+            <IconButton
+                colorScheme="tertiaryElevation0"
+                size="medium"
+                iconName="question"
+                onPress={toggleBottomSheet}
+            />
+            <BottomSheet
```

Comment:
Please use BottomSheetModal instead, we are trying to get rid off the BottomSheet in https://github.com/trezor/trezor-suite/pull/20863

Reply:
can be followup